titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/admission/PassJobValidator.java

```diff
@@ -14,27 +14,26 @@
  * limitations under the License.
  */
 
-package com.netflix.titus.api.jobmanager.model.job.validator;
+package com.netflix.titus.runtime.endpoint.admission;
 
 import java.util.Collections;
 import java.util.Set;
 import javax.inject.Singleton;
 
 import com.netflix.titus.api.jobmanager.model.job.JobDescriptor;
-import com.netflix.titus.common.model.validator.EntityValidator;
-import com.netflix.titus.common.model.validator.ValidationError;
+import com.netflix.titus.common.model.sanitizer.ValidationError;
 import reactor.core.publisher.Mono;
 
 /**
- * This {@link EntityValidator} implementation always causes validation to pass.  It is used as a default implementation which
+ * This {@link AdmissionValidator} implementation always causes validation to pass.  It is used as a default implementation which
  * should be overridden.
  */
 @Singleton
-public class PassJobValidator implements EntityValidator<JobDescriptor> {
+public class PassJobValidator implements AdmissionValidator<JobDescriptor> {
     private final ValidationError.Type errorType;
 
     public PassJobValidator() {
-        this(ValidationError.Type.HARD);
+        this(ValidationError.Type.SOFT);
```

Comment:
Does it matter if it is hard or soft here?

Reply:
I don't think so since this particular implementation never fails validations.